tests/cql/CqlIntervalOperatorsTest.xml

```diff
@@ -3,7 +3,7 @@
 	name="CqlIntervalOperatorsTest" reference="https://cql.hl7.org/09-b-cqlreference.html#interval-operators-3">
 	<group name="After">
 		<test name="TestAfterNull">
-			<expression>null after Interval[1, 10]</expression>
+			<expression>(null as Integer) after Interval[1, 10]</expression>
```

Comment:
Another case where in order to allow the translator to resolve this to one of the signatures, we need to cast `null` to a concrete type to force it.  I also do this in line 100 below.